lib/cadet/course/course.ex

```diff
@@ -57,6 +56,24 @@ defmodule Cadet.Course do
     end
   end
 
+  @doc """
+  Create group entity with given name
+  """
+  def create_group(name, leader, mentor) do
+    %Group{}
+    |> Group.changeset(Map.get(leader, :id), Map.get(mentor, :id), %{name: name})
```

Comment:
You can use `add_belongs_to_id_from_model` as part of the `&Group.changeset/2` at line 20 to allow you to do `%Group{} |> Group.changeset(%{leader: leader, mentor:mentor, name: name})`